website/content/docs/job-specification/check.mdx

```diff
@@ -54,8 +54,7 @@ job "example" {
 - `address_mode` `(string: "host")` - Same as `address_mode` on `service`.
   Unlike services, checks do not have an `auto` address mode as there's no way
   for Nomad to know which is the best address to use for checks. Consul needs
-  access to the address for any HTTP or TCP checks. See
-  [below for details.](#using-driver-address-mode) Unlike `port`, this setting
+  access to the address for any HTTP or TCP checks. Unlike `port`, this setting
```

Comment:
This cointent can still be found at /nomad/docs/job-specification/service#using-driver-address-mode. We should probably link there.